server/index.js

```diff
@@ -33,18 +33,10 @@ export default class Server {
 
   getRequestHandler () {
     return (req, res, parsedUrl) => {
-      if (!parsedUrl) {
+      if (!parsedUrl || parsedUrl && !parsedUrl.query) {
```

Comment:
ok sorry I meant this should probably work: `if (!(parsedUrl && parsedUrl.query))`. But anyways not super important...